gm4_apple_trees/data/gm4_apple_trees/functions/leaf/apple/pick.mcfunction

```diff
@@ -1,9 +1,10 @@
+# update stack count and leaf stage
 # @s = armor_stand, with apple taken from apple tree
 # at player picking the apple
-# run from apple/interact
+# run from gm4_apple_trees:leaf/apple/interact
 
 # return item to player
 item replace entity @a[tag=gm4_apple_take,limit=1] weapon.mainhand from entity @s weapon.mainhand gm4_apple_trees:count
 
 # update armour stand
-execute at @s align xyz run function gm4_apple_trees:tree/leaf/fruiting/set_stage/0
+execute at @s align xyz positioned ~0.5 ~1.5 ~0.5 positioned ^-.25 ^ ^.1 run function gm4_apple_trees:leaf/apple/drop
```

Comment:
Revert this line. It advances to the next stage, with the remaining time added.
`execute at @s align xyz run function gm4_apple_trees:leaf/set_stage/0`